drivers/sdhc/rcar_mmc.c

```diff
@@ -1549,6 +1642,44 @@ static int rcar_mmc_get_card_present(const struct device *dev)
 	return !!(rcar_mmc_read_reg32(dev, RCAR_MMC_INFO1) & RCAR_MMC_INFO1_CD);
 }
 
+/* JESD84-B51, 6.6.5.1 Sampling Tuning Sequence for HS200 */
+static uint8_t tun_block_8_bits_bus[] = {
```

Comment:
Can you make it `const`? I believe, this will benefit microcontrollers like CortexR.

Reply:
done